src/main/target/FLYWOOF411/target.c

```diff
@@ -40,5 +40,5 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM3, CH4, PB1,  TIM_USE_ANY,   0, 0), //  RSSI   1,2
```

Comment:
Why are you making all of these files executable (`100644 → 100755`)? Please revert this.